resources/es/cemu.po

```diff
@@ -2341,7 +2680,7 @@ msgstr ""
 
 #: TitleManager.cpp
 msgid "Irrevocable delete the save entry "
-msgstr "Elimina irrevocablemente la entrada de partida guardada"
+msgstr "Elimina irrevocablemente la entrada de partida guardada "
```

Comment:
The trailing whitespace is a typo of the original string, you can remove it if you want

Reply:
Sure? well, I left it like that because they haven't corrected it yet, and I thought it was for some reason...

Reply:
Yep, that's the tooltip for the "Delete" button of saves